ui/plugins/controller.js

```diff
@@ -397,14 +397,15 @@ treeherder.controller('PluginCtrl', [
                                 const backfilltask = _.find(results.actions, {name: 'backfill'});
                                 // We'll fall back to actions.yaml if this isn't true
                                 if (backfilltask) {
-                                    const actionTask = tcactions.render(backfilltask.task, _.defaults({}, {
-                                        taskGroupId: decisionTaskId,
+                                    return tcactions.submit({
```

Comment:
Yeah, it's used in actions.yml fallback. I'll move it down there to make it more obvious.